pkg/components/azure-arc/image/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mcr.microsoft.com/azure-cli
+
+ARG VERSION
+
+# ENV BASE_URL="https://storage.googleapis.com/kubernetes-helm"
+ENV BASE_URL="https://get.helm.sh"
+ENV TAR_FILE="helm-v${VERSION}-linux-amd64.tar.gz"
+
+
+RUN apk add --update --no-cache curl ca-certificates && \
+    curl -L ${BASE_URL}/${TAR_FILE} |tar xvz && \
+    mv linux-amd64/helm /usr/bin/helm && \
+    chmod +x /usr/bin/helm && \
+    rm -rf linux-amd64 && \
+    apk del curl && \
+    rm -f /var/cache/apk/*
```

Comment:
Maybe just download it using multi-stage build and then copy it over?

Reply:
I think it would be unnecessary as we are till accomplishing the step in a single layer and it seems pretty straightforward.

Reply:
Maybe `mcr.microsoft.com/azure-cli` has `wget` already available, instead of `curl`?